From 84dcb29829e6b4ee85682ded36a727b1b3f11918 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Wed, 10 Jun 2015 11:06:00 -0400
Subject: [PATCH] Fix typo

Follow up for 7c918141ed.

(cherry picked from commit 2fb1105c2bb91c12dd4e66117626da3546afff84)
---
 src/run/run.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/run/run.c b/src/run/run.c
index fcd6b06f7d..d7b1c58455 100644
--- a/src/run/run.c
+++ b/src/run/run.c
@@ -67,7 +67,7 @@ static void help(void) {
         printf("%s [OPTIONS...] {COMMAND} [ARGS...]\n\n"
                "Run the specified command in a transient scope or service or timer\n"
                "unit. If timer option is specified and unit is exist which is\n"
-               "specified with --unit option then command can be ommited.\n\n"
+               "specified with --unit option then command can be omited.\n\n"
                "  -h --help                       Show this help\n"
                "     --version                    Show package version\n"
                "     --user                       Run as user unit\n"
